modules/con-vuln-sources.adoc

```diff
@@ -0,0 +1,70 @@
+* link:https://security-tracker.debian.org/tracker/data/json[Debian Security Tracker]
+* link:https://git.launchpad.net/ubuntu-cve-tracker/[Ubuntu CVE Tracker]
+* link:https://nvd.nist.gov/[NVD]: This is used for various purposes such as filling in information gaps when vendors do not provide information. For example, Alpine does not provide a description, CVSS score, severity, or published date.
++
+[NOTE]
+====
+This product uses the NVD API but is not endorsed or certified by the NVD.
+====
+* link:https://github.com/stackrox/scanner/blob/master/ext/vulnsrc/manual/manual.go and https://github.com/stackrox/scanner/blob/master/pkg/vulnloader/nvdloader/manual.go: The upstream StackRox project maintains a set of vulnerabilities that might not be discovered due to data formatting from other sources or absence of data.
+* link:https://www.redhat.com/security/data/metrics/repository-to-cpe.json[repository-to-cpe.json]: Maps RPM repositories to their related CPEs, which is required for matching vulnerabilities for RHEL-based images.
+
+[id="scanner-v4-vuln-sources"]
+== Scanner V4 sources
+
+Scanner V4 uses the following vulnerability sources:
+
+https://access.redhat.com/security/data/csaf/v2/vex/[Red{nbsp}Hat VEX]:: Used with release 4.6 and later. This source provides vulnerability data in link:https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#45-profile-5-vex[Vulnerability Exploitability eXchange(VEX)] format. {product-title-short} takes advantage of VEX benefits to significantly decrease the time needed for the initial loading of vulnerability data, and the space needed to store vulnerability data.
++
+{product-title-short} might list a different number of vulnerabilities when you are scanning with a {product-title-short} version that uses OVAL, such as {product-title-short} version 4.5, and a version that uses VEX, such as version 4.6. For example, {product-title-short} no longer displays vulnerabilities with a status of "under investigation," while these vulnerabilities were included with previous versions using OVAL data.
++
+For more information about Red Hat security data, including information about the use of OVAL, Common Security Advisory Framework Version 2.0 (CSAF), and VEX, see link:https://www.redhat.com/en/blog/future-red-hat-security-data[The future of Red Hat security data].
+https://access.redhat.com/security/data/metrics/cvemap.xml[Red{nbsp}Hat CVE Map]:: This is used in addition with VEX data for images which appear in the link:https://catalog.redhat.com/software/containers/explore[Red{nbsp}Hat Container Catalog].
+link:https://osv.dev/[OSV]:: This is used for language-related vulnerabilities, such as Go, Java, Node.js (JavaScript), Python, and Ruby. This source might provide
+vulnerability IDs other than CVE IDs for vulnerabilities, such as a GitHub Security Advisory (GHSA) ID.
+
+GitHub Security Advisory (GHSA) IDs rather than CVE numbers for vulnerabilities.
```

Comment:
?

Reply:
Oops, I deleted something but this is a straggler.